cmd/login/login.go

```diff
@@ -60,9 +73,9 @@ func NewLoginCmd() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVar(&username, "username", "", "your user identifier")
-	cmd.Flags().StringVar(&password, "password", "", "your secret password")
-	cmd.Flags().StringVar(&providerID, "provider-id", "", "the authentication provider identifier")
+	cmd.Flags().StringVar(&cfg.Username, "username", "", "your user identifier")
+	cmd.Flags().StringVar(&cfg.Password, "password", "", "your secret password")
+	cmd.Flags().StringVar(&cfg.ProviderID, "provider-id", "", "the authentication provider identifier")
 
```

Comment:
Can we add the --skipCertificate (or --insecure) flag to the `--help` documentation too?

Reply:
Since the `--insecure` flag is global, which should affect all the commands, it is not necessary to add it to a specific command.
Thus, the flag can be already found in the documentation, but under the Global Flags section:

```
> miactl login --help
authenticate with console

Usage:
  miactl login [flags]

Flags:
  -h, --help                 help for login
      --password string      your secret password
      --provider-id string   the authentication provider identifier
      --username string      your user identifier

Global Flags:
      --apiBaseUrl string   api base url
      --apiCookie string    api cookie sid
      --apiKey string       API Key
      --apiToken string     api access token
      --config string       config file (default is $HOME/.miaplatformctl.yaml)
      --insecure            whether to not check server certificate
  -p, --project string      specify desired project ID
  -v, --verbose             whether to output details in verbose mode
  ```

Would you like to insert further documentation in the code?

Reply:
Sorry, you are right, my bad.